src/jit/hwintrinsichelperlistxarch.h

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*****************************************************************************/
+#ifndef SOFTWARE_INTRINSIC
+#error Define SOFTWARE_INTRINSIC before including this file
+#endif
+/*****************************************************************************/
+
+// clang-format off
+
+#ifdef FEATURE_HW_INTRINSICS
+/* Note
+1) Each hardware intrinsic has a unique Intrinsic ID with type of `enum NamedIntrinsic`
+2) All the overloads of an intrinsic in an ISA class share one Intrinsic ID
+3) The helper intrinsic which is implemented in software with use of hardware intrinsics and is part of public API is defined in this table
```

Comment:
Intrinsics which were not in the table with managed implementation were handled by jit as a normal function which due to AggresiveInlining attribute was inlined and intrinsics used in implementation were expanded.

Currently if any of the static classes implement managed intrinsics they have to be included in this table. To make it easier to include native and managed intrinsics tables they were separated into two files.